lib/Game.rb

```diff
@@ -0,0 +1,54 @@
+require_relative "Player"
+require_relative "Board"
+
+class Game
+    def initialize
+        @board =  Board.new
+
+        @player_1 = Player.new("Player 1", "X", @board)
+        @player_2 = Player.new("Player 2", "O", @board)
+
+        @current_player = @player_1
+    end
+
+    def play
+        loop do 
+            @board.show_board
+
+            @current_player.ask_move 
+
+            break if is_game_over?
+
+            @current_player == @player_1 ? @current_player = @player_2 : @current_player = @player_1
```

Comment:
Perhaps a better way to write this is:
`@current_player = @current_player == @player_1 ? @player_2 : @player_1`